handbook/brand.md

```diff
@@ -211,6 +209,13 @@ We approach editing retrospectively for pull requests (PRs) to handbook and docs
 
 > **Note:** The Fleet website may render Markdown differently from GitHub's rich text preview. It's essential to check that PRs merged by the editor are displaying as expected on the site. It can take a few minutes for merged PRs to appear on the live site, and therefore easy to move on and forget. It's good to start the ritual by looking at the site to check that the previous day's edits are displaying as they should.
 
+### How to edit Markdown pull requests for the docs
+
+- When someone creates a pull request for a doc that affects Markdown files, they’ll need to request a review from the editor. 
+- If no edits are needed, the editor will merge the PR. 
+- If an edit changes the meaning, or if unsure, the editor should request a review from the [on-call engineer](https://fleetdm.com/handbook/engineering#oncall-rotation) and remove themselves as a reviewer.
+- If the on-call engineer has questions about the editor’s completed review, they can contact them directly via Slack to resolve these promptly.
```

Comment:
If the on-call engineer has questions, it's better for them to ask directly in the GitHub review. We can just remove this bullet entirely, as I'm confident that commenting on the review will be their go-to action.